example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -150,14 +165,20 @@ describe('The S3 Ingest Granules workflow', () => {
     expectedPayload = loadFileWithUpdatedGranuleIdPathAndCollection(templatedOutputPayloadFilename, granuleId, testDataFolder, newCollectionId);
     expectedPayload.granules[0].dataType += testSuffix;
 
+    // process.env.DISTRIBUTION_ENDPOINT needs to be set for below
```

Comment:
It's completely arbitrary, but I decided that since I'm using a function call to set env vars, that should go in `beforeAll` rather than inside the entire `describe` callback. It seems like we're mostly doing direct assignment inside the `describe` block (besides model instantiation). 

I also kinda liked having this function call near the code using the env vars it creates, but that's also pretty subjective.